extension/package.json

```diff
@@ -5,7 +5,7 @@
     "repository": {
         "url": "https://github.com/Microsoft/vscode-java-test"
     },
-    "version": "0.0.1",
+    "version": "0.1.0",
```

Comment:
Should the internal java package also need changes?

Reply:
java package has already been changed to 0.1.0.

Reply:
already updated